src/alpaca_ast_gen.erl

```diff
@@ -1662,31 +1674,31 @@ invalid_map_type_parameters_test() ->
             "type x = map int",
     Code3 = "module a\n\n"
             "type x = map int int int",
-    ?assertMatch({error,{parse_error,_, {_,{wrong_type_arity,t_map, 0}}}},
+    ?assertMatch({error,{parse_error, ?FILE, 3, {wrong_type_arity,t_map, 0}}},
```

Comment:
See `test_make_modules/1`. `?FILE` is just a builtin erlang macro that gives you the filename of the current file. I just use that as the source of the modules that I define in the tests.